sdk/tests/deqp/framework/common/tcuBilinearImageCompare.js

```diff
@@ -68,13 +68,23 @@ goog.scope(function() {
         return rounded;
     };
 
+    tcuBilinearImageCompare.compareUintRGBA8Threshold = function(a, b, thr) {
+        if (a == b)
+            return true;
+
+        return (Math.abs(((a >> 24) && 0xff) - ((b >> 8) && 0xff)) <= thr.getRed() &&
```

Comment:
Can you implement inline helper function Uint32_R, Uint32_G, Uint32_B, Uint32_A? You can use it here and also in bilinearSampleUintRGBA8.  It makes it easier to read.